### DISCUSSION TITLE
How do I ran a container with a local devlopment taskhub

### DISCUSSION THREAD

**Discussion by StefanGlawing** [microsoft/DurableFunctionsMonitor#96] [Q&A] [answered]
As a part of my development I want to monitor the local taskhub (in Azurite). What is the connection string in this case?

**Comment by scale-tone** [comment, marked as answer]
Hi @StefanGlawing , the so called well-known connection string for Azurite is `UseDevelopmentStorage=true`, or in its expanded form it is:
```
AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;DefaultEndpointsProtocol=http;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;
```

The HTTP ports specified there are the default ones. If for whatever reason you're running Azurite on different ports, then those need to be adjusted accordingly.

If you're using DfMon as a VsCode ext, then it should show you your Azurite-hosted Task Hubs automatically:
![image](https://user-images.githubusercontent.com/5447190/224072508-a04cd4f9-0ae1-43b0-b8d9-10ff5594a1b2.png)

For that, of course, Azurite should be running, and its ports should match the ports in the connection string. By default VsCode ext uses default ports, as above. You can adjust them via this setting:
![image](https://user-images.githubusercontent.com/5447190/224073408-dce4666f-9864-41be-af57-342a850892b9.png)
